libs/application/templates/secondary-school/src/graphql/queries.ts

```diff
@@ -0,0 +1,10 @@
+export const GET_PROGRAMS_BY_SCHOOL_ID = `
```

Comment:
Nitpic about the naming: This is only a string, it's not getting anything. Consider renaming to `PROGRAMS_BY_SCHOOLS_ID_QUERY` or something similar.